tests/backends/aiida_django/test_generic.py

```diff
@@ -1,134 +0,0 @@
-# -*- coding: utf-8 -*-
-###########################################################################
-# Copyright (c), The AiiDA team. All rights reserved.                     #
-# This file is part of the AiiDA code.                                    #
-#                                                                         #
-# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
-# For further information on the license, see the LICENSE.txt file        #
-# For further information please visit http://www.aiida.net               #
-###########################################################################
-"""
-Generic tests that need the use of the DB
-"""
-
-from aiida import orm
-from aiida.backends.testbase import AiidaTestCase
-from aiida.orm import Data
-
-
-class TestComputer(AiidaTestCase):
```

Comment:
So, it seems these tests for computer deletion are no more. Were these already being tested elsewhere and this was redundant?

Reply:
Besides the fact that this is intended to be a backend specific test, yet it only uses the front end classes (making it pointless), this is already tested in the `tests/orm/implementation` tests